ricecooker/commands.py

```diff
@@ -62,7 +66,12 @@ def uploadchannel(chef, command='uploadchannel', update=False, thumbnails=False,
 
     # Get domain to upload to
     config.init_file_mapping_store()
-
+    
+    # Path to chefdata
+    CSV_DIR = os.path.join(Path(__file__).parents[1], 'chefdata', 'data', 'content_metadata.csv')
```

Comment:
This is a full path to a file, not a directory, so we probably want to name this something other than `CSV_DIR`.

Also, `__file__` is the absolute path to `riececooker/commands.py`, not the root of the chef. This will try to save the file inside the Python module. You can use `os.getcwd()` to get the current directory, or even just leave it as a relative path, like you do in `save_metadata_file_as_csv`.

Reply:
updated